zig-mode.el

```diff
@@ -443,9 +443,9 @@ If given a SOURCE, execute the CMD on it."
 	(zig-format-buffer)))
 
 (defun colorize-compilation-buffer ()
-  (toggle-read-only)
+  (read-only-mode)
```

Comment:
```suggestion
  (read-only-mode 0)
```

I think explicitly stating that `read-only-mode` shall be turned off/on is even better.

Reply:
Good idea